internal/gatewayapi/clustersettings.go

```diff
@@ -299,7 +299,11 @@ func buildLoadBalancer(policy egv1a1.ClusterSettings) (*ir.LoadBalancer, error)
 			ConsistentHash: consistentHash,
 		}
 	case egv1a1.LeastRequestLoadBalancerType:
-		lb = &ir.LoadBalancer{}
+		lb = &ir.LoadBalancer{
```

Comment:
should this be ?
```
	case egv1a1.LeastRequestLoadBalancerType:
		lb = &ir.LoadBalancer{
			LeastRequest: &ir.LeastRequest{},
		}
		if policy.LoadBalancer.SlowStart != nil {
			if policy.LoadBalancer.SlowStart.Window != nil {
				lb.LeastRequest.SlowStart = &ir.SlowStart{
				    Window: policy.LoadBalancer.SlowStart.Window,
				}
			}
		}
```		

and same for `RoundRobin` ?

Reply:
I reviewed the logic, this's better.